toolkit/scripts/containerized-build/containerized-build_clean.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Could you double check this still works if you swap between the test/build containers? I think it should be fine but...